api/handler/v1beta1/provider_test.go

```diff
@@ -390,6 +400,16 @@ func (s *GrpcHandlersSuite) TestCreateProvider() {
 							Description: "Please enter your username",
 						},
 					},
+					Policies: []*guardianv1beta1.ProviderPolicy{
```

Comment:
this old test cases should still be passing without having to add this new `policies` field. Because we want to make it backward compatible